modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts

```diff
@@ -166,14 +166,14 @@ export interface SupplementGenerateWalletOptions {
   };
   walletVersion?: number;
   keys: string[];
-  isCold: boolean;
   keySignatures?: {
     backup: string;
     bitgo: string;
   };
   rootPrivateKey?: string;
   disableKRSEmail?: boolean;
   multisigType?: 'tss' | 'onchain' | 'blsdkg';
+  type: 'hot' | 'cold' | 'custodial';
```

Comment:
Please add the `BREAKING CHANGE: <desc>` tag at the end of your commit or else this will not produce the correct sem-ver when releasing the next package.